### ISSUE TITLE
feat: check support forums too

### ISSUE THREAD

**Issue by reisxd** [ReVanced/revanced-discord-bot#62]
## 🐞 Issue

When a member creates a support forum, ReVanced Discord Bot does not check the forums for common issues. 

## ❗ Solution

Listen for forum/thread creation event and join the forum if the content matches.

## ❓ Motivation

This'll save time for the moderation team.

**Comment by oSumAtrIX** [comment]
Currently feature blocked by underlying crate `serenity`

**Comment by Ushie** [comment]
No longer needed
